src/blog/v8-release-84.md

```diff
@@ -0,0 +1,94 @@
+
+### SIMD Origin Trial
+
+The SIMD proposal enables WebAssembly to take advantage of commonly available hardware vector instructions to accelerate compute intensive workloads. V8 has [support](https://v8.dev/features/simd) for the [WebAssembly SIMD proposal](https://github.com/WebAssembly/simd). To enable this in Chrome, use the flag `chrome://flags/#enable-webassembly-simd` or sign up for an [origin trial](https://developers.chrome.com/origintrials/#/view_trial/-4708513410415853567). [Origin trials](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md) allow developers to experiment with a feature before it is standardized, and provide valuable feedback. Once an origin has opted into the trial users are opted into the feature for the duration of the trial period without having to update Chrome flags.
+
+## JavaScript
+
+### Weak references and finalizers
+
+:::note
+**Warning!** Weak references and finalizers are advanced features! They depend on garbage collection behavior. Garbage collection is non-deterministic and may not occur at all.
+:::
+
+JavaScript is a garbage collected language, which means memory occupied by objects that are no longer reachable by the program may be automatically reclaimed when the garbage collector runs. With the exception of references in `WeakMap` and `WeakSet`, all references in JavaScript are strong and prevent the referenced object from being garbage collected. For instance,
+
+```javascript
+const globalRef = {
+  callback() { console.log('foo'); }
+};
+// As long as globalRef is reachable through the global scope,
+// neither it nor the function in its callback property will be collected.
+```
+
+JavaScript programmers can now hold on to objects weakly via the `WeakRef` feature. Objects that are referenced by weak references do not prevent their being garbage collected if they are not also strongly referenced.
+
+```javascript
+const globalWeakRef = new WeakRef({
+  callback() { console.log('foo'); }
+});
+
+(async function() {
+  globalWeakRef.deref().callback();
+  // Logs “foo” to console. Weak ref guaranteed to be alive for the first turn
+  // of the event loop when it is created.
+
+  await (new Promise((resolve, reject) => {
+    setTimeout(() => { resolve('foo'); }, 42);
+  });
+  // Wait for a turn of the event loop.
+
+  globalWeakRef.deref()?.callback();
+  // The object inside globalWeakRef may be garbage collected
+  // after the first turn since it is not otherwise reachable.
+})();
+```
+
+The companion feature to `WeakRef`s is `FinalizationRegistry`, which lets programmers register callbacks to be invoked after an object is garbage collected. For example, the program below may log “42” to the console after the unreachable object in the IIFE is collected.
+
+```javascript
+const registry = new FinalizationRegistry((heldValue) => {
+  console.log(holdings);
```

Comment:
I'm a bit confused by this example - where is `holdings` from? Is it supposed to be `heldValue` given the description above?

Reply:
this should be `heldValue`, updated.